pyproject.toml

```diff
@@ -143,3 +143,23 @@ exclude_lines = [
     "^\\s*raise NotImplementedError$",
     "^\\s*\\.\\.\\.$",
 ]
+
+
+[tool.check-manifest]
+ignore = [
+    # CI
+    ".flake8",
+    ".pre-commit-config.yaml",
+    ".readthedocs.yml",
+    "noxfile.py",
+    # docs
+    "CONTRIBUTING.md",
+    "assets/**",
+    "changelog/**",
+    "docs/**",
+    "examples/**",
+    # tests
+    "test_bot/**",
+    "test_launcher.py",
+    "tests/**",
+]
```

Comment:
Part of the reason for a manifest is to include the source. I feel like we should include a few of these files we're leaving off.

https://discuss.python.org/t/purpose-of-an-sdist/4639

Reply:
There seem to be very differing opinions about this, quoting a few people in that thread:

> So it does not (need to) include any files that aren’t relevant for installing the package, but it also should not contain platform-specific binaries.

> An sdist should also include tests and documentation

> And this is what sdists are to me: whatever is required to build a wheel.

I personally agree with the notion of "an sdist should include everything required to install the package/build a wheel"; the sdist size would increase quite a bit if we were to include documentation (+ images) and tests. The full source including everything is already available from github, which is also linked in the package metadata.

There's also [this](https://discuss.python.org/t/should-sdists-include-docs-and-tests/14578/) post with similar opinions, but ultimately I'd opt for keeping the status quo for now.